dev/devcam/server.go

```diff
@@ -521,7 +525,7 @@ func (c *serverCmd) RunCommand(args []string) error {
 		if c.publish {
 			targets = append(targets, "app/publisher")
 		}
-		targets = append(targets, "app/scanningcabinet")
+		targets = append(targets, "app/pdfcabinet")
```

Comment:
I don't think you meant to remove scanning cabinet here?

Reply:
I did not, thanks for catching it!